### ISSUE TITLE
An error occurred while trying to extend the plugin to add a translatable field

### ISSUE THREAD

**Issue by igor-tv** [rainlab/translate-plugin#592]
I am trying to extend the plugin as in the [documentation](https://octobercms.com/docs/plugin/extending#extending-backend-form) 
```
public function boot()
    {
        Event::listen('backend.form.extendFieldsBefore', function($widget) {
        \Foo\Example\Models\Bar::extend(function($model) {
            $model->translatable[] = 'example_field';
        });
    }
```
And after that I get the error:
```
"Indirect modification of overloaded property \Foo\Example\Models\Bar::$translable has no effect" ...
```
Testing model implements RainLab Translatable behavior.
Working code:
`$model->addDynamicProperty('translatable', ['example_field']);`

**Comment by mjauvin** [comment]
@LukeTowers he opened a second issue which was itself resolved.

**Comment by mjauvin** [comment]
resolved in #593
